mautic/mautic
# New report: Contact's segments activity log

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR added new report type: Contact's segments activity log
This PR allow working with contact event log like date added to segment and date_removed from segment

![image](https://user-images.githubusercontent.com/462477/68783344-2150c500-063b-11ea-9444-f8a939790502.png)

![image](https://user-images.githubusercontent.com/462477/68783639-96bc9580-063b-11ea-886c-42c63048b8ff.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com) + try some contacts add/remove from segments
2. Create report Contact's segments activity log
3. Play with date added to segment  and date removed segment column include filters.
4. Check if your results are correct
